From 3984adc4976de7553f51e0cf4de1e18c373b332b Mon Sep 17 00:00:00 2001
From: Baruch Siach <baruch@tkos.co.il>
Date: Thu, 15 Dec 2016 15:10:19 +0200
Subject: [PATCH] Avoid redefinition of struct ethhdr

This is a workaround to the if_ether.h conflict between musl and the kernel.
Both define struct ethhdr.

Signed-off-by: Baruch Siach <baruch@tkos.co.il>
---
 include/netinet/if_ether.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/include/netinet/if_ether.h b/include/netinet/if_ether.h
index 11ee65823f93..cfe1949d3371 100644
--- a/include/netinet/if_ether.h
+++ b/include/netinet/if_ether.h
@@ -1,6 +1,9 @@
 #ifndef _NETINET_IF_ETHER_H
 #define _NETINET_IF_ETHER_H
 
+/* Suppress kernel if_ether.h header inclusion */
+#define _LINUX_IF_ETHER_H
+
 #include <stdint.h>
 #include <sys/types.h>
 
-- 
2.10.2

